cSpell.json

```diff
@@ -333,6 +255,12 @@
                 "Regenerte"
             ]
         },
+        {
+            "filename": "**/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.1/storage.json",
```

Comment:
Can you add for other keyvault versions too?

Reply:
These were the only ones with the issue